packages/gatsby-theme-store/src/sdk/orderForm/useUpdateQuantityAnalytics.ts

```diff
@@ -0,0 +1,97 @@
+import { useCallback } from 'react'
+
+import { sendPixelEvent } from '../pixel/usePixelSendEvent'
+import type { PixelItem } from '../pixel/events'
+
+export interface UpdateQuantityWithAnalyticsParams<K, S> {
+  updateQuantity: (item: K) => S
+}
+
+export interface OrderFormItemToAnalytics {
+  productId: string
+  productRefId: string
+  /* Product Name */
+  name: string
+  /* Price as integer */
+  price: number
+  quantity: number
+  /* SKU Id */
+  id: string
+  /* SKU Reference Id */
+  refId: string
+  skuName: string
+  additionalInfo: Maybe<{
+    brandName?: Maybe<string>
+  }>
+  productCategories: Record<number, string>
+}
+
+export function orderFormItemToPixelItem(
+  orderFormItem: OrderFormItemToAnalytics
+): PixelItem {
+  return {
+    productId: orderFormItem.productId,
+    productReferenceId: orderFormItem.productRefId,
+    productName: orderFormItem.name,
+    brand: orderFormItem.additionalInfo?.brandName,
+    categoryTree: Object.values(orderFormItem.productCategories).map(
+      (categoryName: string) => ({
+        name: categoryName,
+      })
+    ),
+    price: orderFormItem.price / 100,
+    // TODO currencyCode,
```

Comment:
SHould this go to production?

Reply:
Yes, it should. When we start supporting multiple currencies and etc we should send it with these events, but I thought it was better to not worry about that now.